packages/calypso-e2e/src/element-helper.ts

```diff
@@ -51,6 +51,7 @@ export async function waitForElementEnabled(
 export async function clickNavTab( page: Page, name: string ): Promise< void > {
 	// Mobile view - navtabs become a dropdown.
 	if ( envVariables.VIEWPORT_NAME === 'mobile' ) {
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
While we have not had issues with other NavTabs yet, I felt it would not hurt to add a `networkidle` here. 

I'm not certain of the runtime impact of this statement and so am open to removing this as well.